rest-tests/src/test/java/io/syndesis/qe/CommonHooks.java

```diff
@@ -0,0 +1,15 @@
+package io.syndesis.qe;
+
+import cucumber.api.java.After;
+import io.syndesis.qe.endpoints.TestSupport;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CommonHooks {
```

Comment:
there is already a class defined for the hooks - `io.syndesis.qe.RestTestHooks`

Reply:
I didn't notice that. thx